src/shared/components/structureViewer/StructureViewer.tsx

```diff
@@ -53,10 +63,18 @@ export default class StructureViewer extends React.Component<IStructureViewerPro
             else {
                 this.wrapper.updateViewer(this.props.chainId, this.props.residues, this.props);
             }
+
+            if (!_.isEqual(this.props.bounds, this._bounds)) {
```

Comment:
if I understand correctly what's going on here, you can do this without the extra `_bounds` variable by using the argument to `componentDidUpdate(prevProps)`